log.go

```diff
@@ -394,5 +404,26 @@ func (l *Logger) should(lvl Level) bool {
 	if l.sampler != nil && !samplingDisabled() {
 		return l.sampler.Sample(lvl)
 	}
+
+	if l.section != "" {
+		enabledSections := SectionsProviderFunc()
+
+		for _, entry := range strings.Split(enabledSections, ",") {
+			section := strings.Split(entry, "=")
```

Comment:
Use a parser instead of splitting. This will create a tone of garbage on each execution.

Reply:
Don't you mind if I add a new file called `section.go` which gathers all that stuff related to sections? (parsing, matching,...)

Reply:
No on the contrary. Please do.